databuilder/databuilder/task/neo4j_staleness_removal_task.py

```diff
@@ -64,6 +70,32 @@ class Neo4jStalenessRemovalTask(Task):
 
     """
 
+    delete_stale_nodes_statement = textwrap.dedent("""
+        MATCH (target:{{type}})
+        {staleness_condition}{{extra_condition}}
+        WITH target LIMIT $batch_size
+        DETACH DELETE (target)
+        RETURN COUNT(*) as count;
+        """)
+    delete_stale_relations_statement = textwrap.dedent("""
+        MATCH (start_node)-[target:{{type}}]-(end_node)
+        {staleness_condition}{{extra_condition}}
+        WITH target LIMIT $batch_size
+        DELETE target
+        RETURN count(*) as count;
+        """)
+    validate_node_staleness_statement = textwrap.dedent("""
+        MATCH (target:{{type}})
+        {staleness_condition}{{extra_condition}}
+        WITH DISTINCT labels(target) as node, count(*) as count
+        RETURN head(node) as type, count
+        """)
+    validate_relation_staleness_statement = textwrap.dedent("""
```

Comment:
why are we still returning type here

Reply:
Users have the ability to override the allowed staleness percentage for types in the config which is why I left it originally, but you're right that I'm not using it from the query since we have the types now so I'll take it out